### PR TITLE
Tee stdout to a log file as the disk image is being built (v1.3.0-rc)

### PR DESCRIPTION
**Pull request by brtkwr**

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -89,12 +89,14 @@
...
       {% if item.size is defined %}--image-size {{ item.size }}{% endif %}
       {% if item.type is defined %}-t {{ item.type }}{% endif %}
       {{ os_images_common }}
-      {{ item.elements | join( ' ' ) }} -o {{ item.name }}
+      {{ item.elements | join( ' ' ) }}
+      -o {{ item.name }}
+  shell: "{{ os_images_venv }}/bin/disk-image-create {{ dib_args }} | tee {{ item.name }}.stdout"
   args:
     chdir: "{{ os_images_cache }}/{{ item.name }}"
     creates: "{{ os_images_cache }}/{{ item.name }}/{{ item.name }}.d/dib-manifests"
```
Comment (markgoddard) [inline tasks/main.yml:102]: Do we still need the following task if we're teeing to a file? Could we just redirect stdout and stderr to files?

command > file.stdout 2> file.stderr

File: tasks/main.yml [hunk 2]
```diff
@@ -89,12 +89,14 @@
...
-    {{ os_images_venv }}/bin/disk-image-create
+  vars:
+    dib_args: >-
       {% if item.size is defined %}--image-size {{ item.size }}{% endif %}
       {% if item.type is defined %}-t {{ item.type }}{% endif %}
       {{ os_images_common }}
-      {{ item.elements | join( ' ' ) }} -o {{ item.name }}
+      {{ item.elements | join( ' ' ) }}
+      -o {{ item.name }}
+  shell: "{{ os_images_venv }}/bin/disk-image-create {{ dib_args }} | tee {{ item.name }}.stdout"
```
Comment (markgoddard) [inline tasks/main.yml]: Do we need stderr too?

File: tasks/main.yml [hunk 3]
```diff
@@ -109,21 +109,6 @@
   when: item.elements is defined
   no_log: true
```
Comment (markgoddard) [inline tasks/main.yml]: We should be able to remove no_log now - this was to avoid the output dump.

File: tasks/main.yml (hunk 1 above)
Comment (brtkwr) [reply tasks/main.yml:102]: Good tip!

File: tasks/main.yml (hunk 2 above)
Comment (brtkwr) [reply tasks/main.yml]: Sounds good.

File: tasks/main.yml (hunk 3 above)
Comment (brtkwr) [reply tasks/main.yml]: Done

**Review by markgoddard — APPROVED** [review]

**Pull request merged**